eclipse-mosquitto.yaml

```diff
@@ -0,0 +1,110 @@
+package:
+  name: eclipse-mosquitto
+  version: 2.0.18
+  epoch: 0
+  description: Eclipse Mosquitto - An open source MQTT broker
+  copyright:
+    - license: EPL-2.0 AND BSD-3-Clause
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+      - cjson
+      - libwebsockets-dev
+      - mosquitto
+      - mosquitto-clients
+      - mosquitto-dev
+
+var-transforms:
+  - from: ${{package.version}}
+    match: ^(\d+\.\d+)\.\d+$
+    replace: "$1"
+    to: major-minor-version
+
+environment:
+  contents:
+    packages:
+      - busybox
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 3923526c6b4c048bbecad2506c4c9963bc46cd36
+      repository: https://github.com/eclipse/mosquitto
+      tag: v${{package.version}}
+
+  - runs: |
+      mkdir -p ${{targets.contextdir}}
+      mkdir -p ${{targets.contextdir}}/mosquitto/config
+      mkdir -p ${{targets.contextdir}}/mosquitto/data
+      mkdir -p ${{targets.contextdir}}/mosquitto/log
+      install -Dm644 docker/${{vars.major-minor-version}}-openssl/docker-entrypoint.sh ${{targets.contextdir}}/docker-entrypoint.sh
+      install -Dm644 docker/${{vars.major-minor-version}}-openssl/mosquitto-no-auth.conf ${{targets.contextdir}}/mosquitto-no-auth.conf
+
+update:
+  enabled: true
+  github:
+    identifier: eclipse/mosquitto
+    strip-prefix: v
+    use-tag: true
+
+test:
+  environment:
+    contents:
+      packages:
+        - libwebsockets-dev
+        - mosquitto-dev
```

Comment:
Same as above about dev package at runtime, is it needed?

Reply:
yep its needed.